template/listentry/template.proto

```diff
@@ -22,6 +22,21 @@ option (istio.mixer.v1.config.template.template_variety) = TEMPLATE_VARIETY_CHEC
 
 // ListEntry is used to verify the presence/absence of a string
 // within a list.
+//
+// When writing the configuration, the value for the fields associated with this template can either be a
+// constant literal or an [expression](https://istio.io/docs/reference/config/mixer/expression-language.html). Please note that if the datatype of a field is not [VALUETYPE](https://github.com/istio/api/blob/master/mixer/v1/config/descriptor/value_type.proto),
```

Comment:
suggest to use a branch, not master.

Reply:
Thanks, removed, since operator does not care out the details of the ValueType and ValueType will be defined right above the templates in the docs.